packages/core/src/components/Modal/__stories__/Modal.mdx

```diff
@@ -53,11 +53,7 @@ Use modal for short and non-frequent tasks. For common tasks consider using the
 
 ### Sizes
 
-<Canvas>
-  <Story of={ModalStories.WidthVariantsNormal} />
-  <Story of={ModalStories.WidthVariantsFull} />
-  <Story of={ModalStories.WidthVariantsCustom} />
-</Canvas>
+<Canvas of={ModalStories.Sizes} />
```

Comment:
`Sizes` does not exist on `ModalStories`.
This can be seen when you'll run storybook and this would fail to preset the "Modal" page.

Please take a look at the "Button" story, as stated in the issue #2500:
> See Button's story for ["Sizes"](https://github.com/mondaycom/vibe/blob/master/packages/core/src/components/Button/__stories__/Button.mdx?rgh-link-date=2024-10-13T07%3A10%3A10Z#sizes)
And its appropriate [story source](https://github.com/mondaycom/vibe/blob/master/packages/core/src/components/Button/__stories__/Button.stories.tsx?rgh-link-date=2024-10-13T07%3A10%3A10Z#L53)

You should do the same here